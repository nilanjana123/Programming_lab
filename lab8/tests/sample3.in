#dxa
# This line is a comment
# This file has a scalar (123), and a 2 x 3 matrix (elements 1 ... 6)

2

3 3
2,4,6,
8,10,12,
14,16,18;

2 2
1,2,
6,9,

