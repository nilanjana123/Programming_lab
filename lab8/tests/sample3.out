# cmd options :dxa
# After division.
3 3
1,2,3,
4,5,6,
7,8,6,

# After determinant.
27

# After addition.
2 2
28,29,
33,36,
